iqm5q/platform.py

```diff
@@ -9,22 +9,22 @@
 from qibolab.instruments.dummy import DummyLocalOscillator
 from qibolab.instruments.rohde_schwarz import SGS100A
 from qibolab.instruments.zhinst import Zurich
+from qibolab.kernels import Kernels
 from qibolab.serialize import (
     load_instrument_settings,
     load_qubits,
     load_runcard,
     load_settings,
 )
 
-RUNCARD = pathlib.Path(__file__).parent / "iqm5q.yml"
-FOLDER = pathlib.Path(__file__).parent / "iqm5q/"
+FOLDER = pathlib.Path(__file__).parent
 
 TWPA_ADDRESS = "192.168.0.35"
 
 N_QUBITS = 5
 
 
-def create(runcard_path=RUNCARD):
+def create(folder: pathlib.Path = FOLDER):
```

Comment:
Following https://github.com/qiboteam/qibolab/pull/758#discussion_r1467268557, I am not sure if passing the path as an argument to these functions has any use case. Before it could be used to load the runcard YAML from a different place, that's not very relevant anymore given that the runcard lies in a folder with the `create`. In practice, if someone wants to load a different runcard it is simpler to create a new platform folder (with new create+runcard) altogether.

@alecandido @andrea-pasquale what do you think?

The suggestion is to drop `folder` from argument and just use the constant `FOLDER` directly in the body.

Reply:
Agreed.

I believe the mentioned option has never been used, and if you want to have multiple parameters (e.g. different stages in calibration, or previous calibrations...), you could keep them all in the folder, and just rename the one you want to use.

Reply:
Fine also with me.